docs/claims.md

```diff
@@ -0,0 +1,175 @@
+
+If you are making a claim on Polkadot for the first time please read on below in the
+[making a claim](#making-a-claim) section. If you've already claimed during the pre-genesis claims
+please proceed to [attesting to a statem](#attesting-to-a-statement) section instead.
+
+## Making a Claim
+
+If you did not make a claim in the pre-genesis claims period then you are able to make a claim any
+time after genesis. There is no time limit for making your claim so feel free to do this whenever
+you are most comfortable.
+
+> Note: When you make a claim, at the same time you will make an attestation to the required
+> statement. The two actions "claim" and "attest" are done at once in a `claim_attest` transaction,
+> but for the most part this is simplified from the perspective of the UI.
+
+### What you will need
+
+- The Ethereum account that holds the DOT indicator tokens.
+- The MyCrypto wallet.
+- A Polkadot account.
+
+You should already have your Ethereum account that holds the DOT indicator tokens from Polkadot's
+prior sales. You will need to have access to this account in order to make a signature.
+
+[MyCrypto][] is a versatile wallet that supports a variety of storage methods for your Ethereum
+account. Please click on the link to go to their downloads page and ensure you download the latest
+version for your operating system. This is important because the latest version will always have the
+latest security patches.
+
+You will need a Polkadot account to mark as the destination for sending the DOTs in the claim.
+Please follow the instructions on the [account generation][] page for generating a new Polakdot
+account.
+
+#### Claiming your DOT with MyCrypto
+
+The Polkadot JS [Claims app][] helps you sign a message from MyCrypto. MyCrypto is a good choice in
+case you have stored the key to the Ethereum account holding your DOT indicator tokens on a hardware
+device like a Ledger Nano S or a Trezor. It also supports raw private keys, mnemonics and Parity
+signer.
+
+**NOTICE**: It is much more secure to download and use the MyCrypto app locally. Please make sure to
+download the latest version for your operating system. You can always find the most up-to-date
+releases of the desktop app on their [releases page][mycrypto].
+
+Once you've downloaded MyCrypto and have it running locally (we recommend an air-gapped computer for
+maximum security), you can start by navigating to the Claims app on Polkadot-JS Apps. Select the
+account you would like to claim the DOTs into and click the blue "Continue" button to proceed. Your
+screen should look something like this:
+
+![claim-1](assets/new-claims/claim-1.png)
+
+Now you will need to provide the Ethereum address that is associated with the DOT indicator tokens
+that you will claim. Enter the Ethereum address into the box and click "Continue". Next your screen
+should look the image below.
+
+![claim-2](assets/new-claims/claim-2.png)
```

Comment:
Just a heads-up that the latest version includes the correct payload to sign and the full statement:

<img width="641" alt="Screenshot 2020-05-25 at 11 24 45" src="https://user-images.githubusercontent.com/1293565/82799497-6e2f1700-9e7a-11ea-8ea4-434b22150dd3.png">

Reply:
Lots of typos in the above screenshot. Where can one fix this?

Reply:
https://github.com/polkadot-js/apps/pull/2734

But the screenshot is already outdated, so I think the typos may have been fixed.